### PR TITLE
[stable/sonatype-nexus] Add annotation configuration in pvc

### PR DESCRIPTION
**Pull request by rodesousa**
I would like to keep a pvc object when I delete my release.
I use `"helm.sh/resource-policy": keep` in this case

### CODE REVIEW

**Comment by unguiculus** [comment]
cc @rjkernick

File: stable/sonatype-nexus/README.md [hunk 1]
```diff
@@ -54,7 +54,7 @@ The following tables lists the configurable parameters of the Nexus chart and th
 | `service.readinessProbe.periodSeconds`      | Seconds between polls               | 30                                         |
 | `service.readinessProbe.failureThreshold`   | Number of attempts before failure   | 6                                          |
 | `service.livenessProbe.initialDelaySeconds` | livenessProbe initial delay         | 30                                         |
-| `service.livenessProbe.periodSeconds`       | Seconds between polls               | 30                                         |                
+| `service.livenessProbe.periodSeconds`       | Seconds between polls               | 30                                         |
 
 Specify each parameter using the `--set key=value[,key=value]` argument to `helm install`. For example,
```
Comment (rjkernick) [inline stable/sonatype-nexus/README.md:61]: Can you please update the readme to include the new pvc annotation setting?

Comment (rodesousa) [reply stable/sonatype-nexus/README.md:61]: done

**Comment by rodesousa** [comment]
miss something ?

**Pull request merged**